web/src/backend/controllers/asset.js

```diff
@@ -29,13 +29,13 @@ export const createAssetController = async (req, res) => {
       .status(HANDLED_ERROR_RESPONSE)
       .json({ message: validateResult.message });
   }
-  let { result, status } = await createAsset({
```

Comment:
Remove this causes more error
![image](https://user-images.githubusercontent.com/46278271/100062882-e646bb80-2e62-11eb-833c-ddfbd748b4fb.png)